src/components/Footer.jsx

```diff
@@ -10,7 +11,7 @@ const Footer = () => (
     <div className={layout.sectionReverse}>
       <div className={`px-6 ${layout.sectionInfo} md:ml-16`}>
         <h2 className="text-xl font-bold text-gray-800 font-poppins dark:text-white hover:text-gray-700 dark:hover:text-gray-300">
-          Parth Mittal
+          {aboutMe[0].name} {aboutMe[0].lastName}
```

Comment:
Change here as well according to `aboutMe` object.

Reply:
done!